test/steps/1_given.js

```diff
@@ -751,6 +812,34 @@ export function aConfigWithJsonSetTo() {
 	this.test.ctx.config = config;
 }
 
+export function anOptionsObjectWithPassphraseSetToAndSecondPassphraseSetToStdIn() {
+	const [passphraseSource, secondPassphraseSource] = getQuotedStrings(this.test.parent.title);
+	const { passphrase } = this.test.ctx;
```

Comment:
Prefer destructuring out of `test.ctx` as the first line of step definition functions.